backend/terraform/modules/analyzer/db.tf

```diff
@@ -64,6 +53,70 @@ resource "aws_rds_cluster_parameter_group" "cluster_parameter_group" {
   tags = var.tags
 }
 
+resource "random_password" "rds_master_password" {
+  length  = 10
+  special = false
+  numeric = true
+}
+
+resource "null_resource" "aurora_db_master_password" {
+  depends_on = [
+    aws_secretsmanager_secret_version.db-master-password
+  ]
+  triggers = {
+    db_host = aws_rds_cluster.aurora.endpoint
+  }
+  provisioner "local-exec" {
+    # Pull the actual DB master password and set it
+    command = "bash -c 'DBPASS=$(aws --region ${var.aws_region} secretsmanager get-secret-value --secret-id ${var.app}/db-master-password | jq -r \".SecretString\"); aws --region ${var.aws_region} rds modify-db-cluster --db-cluster-identifier ${aws_rds_cluster.aurora.id} --master-user-password \"$${DBPASS}\" --apply-immediately'"
+  }
+}
+
+resource "random_id" "rds_snapshot_identifier" {
+  byte_length = 4
+  keepers = {
+    "id" = "${local.db_prefix}"
+  }
+}
+
+############################ 
+# Aurora Cluster Instance ##
+############################
+resource "aws_rds_cluster_instance" "cluster_instance" {
+  count = 1 # Create 1 db instance
+
+  apply_immediately          = true
+  auto_minor_version_upgrade = true
+  availability_zone          = element(var.database_availability_zones, length(var.database_availability_zones) - 1)
```

Comment:
To support more than a single instance (i.e., `count` > 1), we'll want to spread the instances across the available AZs.

I'm assuming we're picking the last element in the list to match what's currently deployed?  If so, we could do something like:

```terraform
availability_zone = element(reverse(var.database_availability_zones), count.index)
```